src/runtime/test/suggestion-composer-tests.js

```diff
@@ -86,15 +84,22 @@ describe('suggestion composer', function() {
     assert.isEmpty(suggestionComposer._suggestConsumers);
 
     await helper.acceptSuggestion({particles: ['List', 'CakeMuxer']});
-    await helper.makePlans();
+    await helper.makePlans({includeInnerArcs: true});
     assert.lengthOf(helper.suggestions, 1);
     await suggestionComposer.setSuggestions(helper.suggestions);
     assert.lengthOf(suggestionComposer._suggestConsumers, 1);
     const suggestConsumer = suggestionComposer._suggestConsumers[0];
     await suggestConsumer._setContentPromise;
     assert.isTrue(suggestConsumer._content.template.includes('Light candles on Tiramisu cake'));
 
-    await helper.acceptSuggestion({particles: ['LightCandles']});
+    // TODO(mmandlis): Better support in test-helper for instantiating suggestions in inner arcs.
+    // Instantiate inner arc's suggestion.
+    const innerSuggestion = helper.findSuggestionByParticleNames(['LightCandles'])[0];
+    const innerArc = helper.arc.innerArcs[0];
+    // This is a hack: inner arc suggestion's particle spec part of the inner arc context.
+    innerArc._context = helper.arc.context;
+    await innerSuggestion.instantiate(innerArc);
```

Comment:
Up until now `Suggestion` object didn't need an arc reference, because suggestions are stored per arc, so only suggestions of the currently active arc are consumed at any given point of time.

Possibly we need to add an arc ID to the suggestion serialization - it will come handy once we strategize for inner arcs and end up with a set of suggestions for different arcs.

As an interim solution, it is easy to update test-helper to store a map of suggestions-by-arc, but I prefer to wait until we implement the strategizing for inner arcs and see where it brings us, if that sounds reasonable.